components/TradingView/index.tsx

```diff
@@ -29,11 +32,21 @@ export interface ChartContainerProps {
   theme: string
 }
 
-// export interface ChartContainerState {}
+const lines = new Map()
+let markPrice = 0
```

Comment:
I don't see a reason why this shouldn't be solved with the useState hook. Or am I missing something?